pkg/userdata/rhel/rhel.go

```diff
@@ -28,6 +28,10 @@ type Config struct {
 	RHELSubscriptionManagerUser     string `json:"rhelSubscriptionManagerUser,omitempty"`
 	RHELSubscriptionManagerPassword string `json:"rhelSubscriptionManagerPassword,omitempty"`
 	RHSMOfflineToken                string `json:"rhsmOfflineToken,omitempty"`
+	UseRHSatelliteServer            bool   `json:"useRHSatelliteServer"`
+	RHSatelliteServer               string `json:"rhSatelliteServer"`
+	OrganizationName                string `json:"organizationName"`
+	ActivationKey                   string `json:"activationKey"`
```

Comment:
I see nice patter here to prefix RHEL specific fields with `RHEL`, let's continue to do so (including in JSON)

Reply:
yeah I agree.